### DISCUSSION TITLE
Page Up/Dn Keys

### DISCUSSION THREAD

**Discussion by jasondh** [qutebrowser/qutebrowser#6779] [General] [answered]
Using version 2.4.0 on Mac (M1) OS 12.0.1.

I see that my Page Up and Page Dn keys no longer work on any web page.  Same for Cursor keys.

Anyone else getting this?

**Comment by The-Compiler** [comment, marked as answer]
How do things look if you start with `--temp-basedir`? You should be able to do so via something like `/Applications/qutebrowser.app/Contents/MacOS/qutebrowser --temp-basedir` (path from memory, might not be 100% correct).

**Reply by jasondh** [reply]
Ah - so that works fine when running from the command line with _--temp-basedir_

Can you advise what I need to do next to get this working?  Cheers.

**Reply by jasondh** [reply]
[Updated] - I have deleted the config.py file and is back to working.
